pinot-segment-spi/src/test/java/org/apache/pinot/segment/spi/index/creator/H3IndexConfigTest.java

```diff
@@ -84,4 +84,16 @@ public void withSomeData()
     Assert.assertEquals(resolution.getLowestResolution(), 5);
     Assert.assertEquals(resolution.getResolutions(), Lists.newArrayList(5, 6, 13));
   }
+
+  @Test
+  public void serde()
+      throws JsonProcessingException {
+    H3IndexConfig initialConf = new H3IndexConfig(new H3IndexResolution(Lists.newArrayList(5, 6, 13)));
+
+    String confAsJson = JsonUtils.objectToString(initialConf);
+    Assert.assertEquals(confAsJson, "{\"disabled\":false,\"resolution\":[5,6,13]}");
```

Comment:
can we support both 
`\"resolution\":[5,6,13]` and `\"resolution\":8663` (which is the bit-shifted version of the array)

Reply:
We would need to add a custom serializer and deserializer, but it is possible.

Reply:
The new version supports what you said (although the bit shifted version seems to be `8288`).

But we need to choose which representation are we going to use when serializing. I used the string version, but we can change that we prefer to use the int one.